static/js/giphy.js

```diff
@@ -99,6 +108,44 @@ function update_grid_with_search_term() {
     return undefined;
 }
 
+export function hide_giphy_popover() {
+    if (active_popover_element) {
+        active_popover_element.popover("hide");
+        active_popover_element = undefined;
+        return true;
+    }
+    return false;
+}
+
+function get_popover_template() {
+    let template = render_giphy_picker();
+
+    if (window.innerWidth <= 768) {
```

Comment:
Should this be one of our size constants?

Reply:
Yes.

Reply:
Yes.